.github/workflows/ci.yml

```
@@ -107,13 +114,15 @@ jobs:
       - name: Install uv
         uses: astral-sh/setup-uv@v3
 
-      - name: Restore cached dependencies
+      - name: Cache uv dependencies
         uses: actions/cache@v3
```

actionlint (1.7.9)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)